magnus

```diff
@@ -56,6 +57,14 @@ class Main(object):
             print("    Show about dialogue")
             print("  --refresh-interval=120")
             print("    Set refresh interval in milliseconds (lower is faster)")
+            print("  --force-refresh")
+            print("    Refresh every on interval even if the viewport has not moved.")
```

Comment:
Typo here, should be "refresh on every ..." Sorry. Feel free to rewrite the help message or comments.